### PR TITLE
Add pub/sub to simple RabbitMQ sample

### PR DESCRIPTION
**Pull request by timbussmann**
Splits the single endpoint that used to `SendLocal` into the common Sender/Receiver/Shared structure that is quite common on our samples and let the user choose between sending commands or publishing events.

### CODE REVIEW

File: samples/rabbitmq/simple/Rabbit_5/Receiver/MyEventHandler.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    using NServiceBus;
+    using NServiceBus.Logging;
+
+    public class MyEventHandler : IHandleMessages<MyEvent>
+    {
+        static ILog log = LogManager.GetLogger<MyEventHandler>();
+
+        public Task Handle(MyEvent eventMessage, IMessageHandlerContext context)
+        {
+            log.Info($"Hello from {nameof(MyEventHandler)}");
```
Comment (SzymonPobiega) [inline samples/rabbitmq/simple/Rabbit_5/Receiver/MyEventHandler.cs:13]: Can we rename `MyEventHandler` to `TheOtherSide` ? 😂

File: samples/rabbitmq/simple/Rabbit_5/Sender/Program.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        var endpointInstance = await Endpoint.Start(endpointConfiguration)
+            .ConfigureAwait(false);
+        await SendMessages(endpointInstance);
+        await endpointInstance.Stop()
+            .ConfigureAwait(false);
+    }
+
+    static async Task SendMessages(IMessageSession messageSession)
+    {
+        Console.WriteLine("Press [c] to send a command, or [e] to send an event. Press [Esc] to exit.");
```
Comment (SzymonPobiega) [inline samples/rabbitmq/simple/Rabbit_5/Sender/Program.cs:29]:
```suggestion
        Console.WriteLine("Press [c] to send a command, or [e] to publish an event. Press [Esc] to exit.");
```

**Review by SzymonPobiega — APPROVED** [review]

File: samples/rabbitmq/simple/Rabbit_5/Sender/Program.cs (hunk 2 above)
Comment (timbussmann) [reply samples/rabbitmq/simple/Rabbit_5/Sender/Program.cs:29]: ugh, good catch, thanks 👍

**Pull request merged**